ui/app/styles/clinical/treatment/_treatment.scss

```diff
@@ -169,6 +169,7 @@
             }
         }
 
+        button.ipd-btn,
```

Comment:
@Phanindra-tw The IPD feature is still under discussion, and the changes added here to support it are not relevant to this PR. Please revert them.

cc: @binduak, @angshu